### PR TITLE
RST-2707 - test updates

### PR DESCRIPTION
**Pull request by roortest**
Replaced Phantomjs and Poltergeist with Apparition
updated env file
Gem & Gemlock files

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 # require "action_cable/engine"
-require "sprockets/railtie"
+ require "sprockets/railtie"
```
Comment (zaparka) [inline config/application.rb:12]: Can you remove this space please?

File: yarn.lock [hunk 2]
```diff
@@ -2,7 +2,6602 @@
 # yarn lockfile v1
```
Comment (zaparka) [inline yarn.lock:2]: @roortest  I'm not sure why there is so much "dependencies" in this file but I think it's not correct. I removed this file on my local and used yarn to generated and I got just the govuk-frontend dependecy. Which I think is correct. Can you try on you local please?

File: config/application.rb (hunk 1 above)
Comment (zaparka) [reply config/application.rb:12]: And the commented out line too please.

**Review by zaparka — CHANGES_REQUESTED** [review]

File: config/application.rb (hunk 1 above)
Comment (roortest) [reply config/application.rb:12]: okay sure @zaparka

File: yarn.lock (hunk 2 above)
Comment (roortest) [reply yarn.lock:2]: that is due to web packer installation  i did it before to resolve those assets issues 
removed and reinstalled yarn . every thing sorted now. thanks .

File: config/application.rb (hunk 1 above)
Comment (roortest) [reply config/application.rb:12]: updted

**Review by zaparka — APPROVED** [review]

**Pull request merged**